mailpoet/lib/WooCommerce/Subscription.php

```diff
@@ -252,6 +254,28 @@ public function handleSubscriberOptin(SubscriberEntity $subscriber, bool $checko
     return true;
   }
 
+  private function hideAutomateWooOptinCheckbox() {
+    if (!$this->wp->isPluginActive('automatewoo/automatewoo.php')) {
+      return false;
```

Comment:
The return value from this method is never used. I guess we can use `return` and declare the return type to `void`. 
The same approach can be used for the following method.
What do you think?

Reply:
Sure, we can do that, I have made the change.